### ISSUE TITLE
view raw error

### ISSUE THREAD

**Issue by pinghe** [takezoe/gitbucket#930]
~~~~
org.eclipse.jgit.errors.LargeObjectException: ae71c886bc48faa455c591f5199d059be01231b0 exceeds size limit
	at org.eclipse.jgit.internal.storage.file.LargePackedWholeObject.getCachedBytes(LargePackedWholeObject.java:98)
~~~~

**Comment by nazoking** [comment]
maybe this is same as #933

**Comment by takezoe** [comment]
Merged with #933

**Comment by pinghe** [comment]
yes， same as #933.

i need store release version. binaries over 100MB
